main/index.html

```diff
@@ -163,12 +163,12 @@ <h1 class="title1">MetWork Framework</h1>
                 </div>
                 <!-- layer 2 -->
                 <div class="layer-1-2 wow slideInUp" data-wow-duration="2s" data-wow-delay=".1s">
-                  <h2 class="title2">Build your own system with our "battle tested" software bricks</h2>
+                  <h2 class="title2">Build production ready systems with "battle tested" software bricks</h2>
```

Comment:
Because of previous comment:

```suggestion
                  <h2 class="title2">Save time in building, running and monitoring your "production ready" applications by using our "battle tested" software bricks</h2>
```

Reply:
Ok I close this thread to open another one